release-notes/major9/09.25.0/valtimo-frontend-libraries.md

```diff
@@ -0,0 +1,21 @@
+# Frontend libraries 5.14.0 (RC)
+
+## New Features
+
+No features added
+
+## Bugfixes
+
+No bugfixes.
+
+## Breaking changes
+
+No breaking changes.
+
+## Deprecations
+
+No new deprecations.
+
+## Known issues
+
+No new known issues.
```

Comment:
I should use the templates found [here](https://github.com/valtimo-platform/valtimo-documentation/tree/main/release-notes/template). The templates contains the layout that should be used for these files.

The switch to the "No new ...." texts is only done when the release is build.

This should also be changed in valtimo-backend-libraries.md and please add the migration.md

Reply:
Fixed. Also updated the index and removed a dead link to release-notes/major9/09.20.0/migration.md